### DISCUSSION TITLE
Sometimes getting empty request to livewire component

### DISCUSSION THREAD

**Discussion by marispro** [livewire/livewire#4262] [1. Help]
Hello,
I am using livewire latest version, but this issue was actual from whole livewire beginnings.

Sometimes I am getting/seeing this in error logs:
`ErrorException (E_WARNING) Trying to access array offset on value of type null`
`livewire/livewire/src/LivewireManager.php (line: 372)`

This happens when server gets broken xhr request to the component with empty body.
No idea who exactly might do this, probably crawlers or somebody. But this becoming annoying.

Can we add check for LivewireManager to check the existing of fingerprint url/path? Or someone have better idea how to get rid of this?

Thanks.

**Comment by joshhanley** [comment]
@marispro yeah that's not ideal! PR's are welcome if you want to add a check in 🙂

Thanks!

**Comment by marispro** [comment]
PR: https://github.com/livewire/livewire/pull/4316
